### ISSUE TITLE
Dung with Hatched Egg doesn't decay

### ISSUE THREAD

**Issue by gsamsom** [jasonrohrer/OneLifeData7#402]
Where normal dung decays in 1800 seconds, Dung with Hatched Egg doesn't.

Suggested fix:
Dung with Hatched Egg decays to normal Dung in 20-60 seconds

or:
Dung with Hatched Egg decays in 1800 seconds

**Comment by laccy** [comment]
Actually this is a good way to preserve dung if needed, quite a few of us doing this for this reason.

Dung might need another use, compost might need a buff, some foods would need a higher soil/lower water usage. Now with all the stored dung we never use it up but pls stop nerfs and add sinks. Also while sheared sheep can poop and people refuse to feed lambs only, sheared sheep shouldn't be able to be fed. Maybe add another stage, so people need to sheer 2x and would only poop when grown-up, and only be able to kill on the third stage when it's old.

As the player who made the most pens in history of OHOL, it's quite annoying to see that people either fill a big pen with garbage and don't clean it out, or feed sheep instead of lambs and all that meat is wasted. I still saw way too much mutton meat at times, so another stage would help.
1 lamb
2 young sheep can give wool, won't poop, won't have lamb, can't be killed for meat
3 old sheep, poops, can be slaughtered

the only issue would be if young sheep escapes pen, but since we got ropes to move them, it's not that bad
it would increase the fleece for clothes, save up for space and reduce meat amount.

I tried a setup for butchering sheep in separate zone, people didn't really understand it.
Basically made tables to block them, moved sheared sheep with a rope since they won't poop or have lambs and they are biome locked. This saved up space in pen and forced people to feed lambs instead. It would still need support so a butchering area is a thing that players make.
The best thing to change that would be a point where they can't get any more wool of the sheep and the only thing they are useful is meat. So doubling fleece but reducing the usefulness of the sheep would require more work to get fleece but less waste of food and less poop overflood.
